l10n_nl_tax_declaration_reporting/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+=========================
+NL Tax Declaration Report
+=========================
+
+This module provides you with the Tax Statement in the Dutch format.
+In the report you will find the taxes charged to customers and taxes that
+suppliers charge to you. The layout this module provides will seem very
+familiar to everyone wanting to declare taxes in The Netherlands.
+
+The standard Tax Declaration report is not replaced by this module, a new
+report with the Dutch layout is added to the menu
+Reporting > Generic Reporting > Taxes.
+
+Deze module biedt u de BTW aangifte in Nederlands formaat.
+In het rapport staat de BTW berekend aan klanten, en de btw die
+leveranciers aan u berekenen. Het rapport is met deze indeling zeer
+herkenbaar voor iedereen die een belasting aangifte wilt doen.
+
+Dit rapport verwijdert niet de standaard, maar wordt toegevoegd in
+het menu Rapportages > Algemene rapporten > Belastingen.
+
+Installation
+============
+
+This module can be installed using the standard installation procedure.
+
+Configuration
+=============
+
+No configuration is needed.
+
+Usage
+=====
+
+The standard Tax Declaration report is not replaced by this module, a new
+report with the Dutch layout is added to the menu
+Reporting > Generic Reporting > Taxes.
+
```

Comment:
You can add a 'try me on runbot' stanza. See https://github.com/OCA/l10n-netherlands/pull/19/files#diff-59213b219279126ed83412ae4953619cR43 for an example that you can just cut and paste.